SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -130,7 +130,7 @@ message SyncMessage {
   }
 
   message Contacts {
-    optional AttachmentPointer blob = 1;
+    optional AttachmentPointer blob     = 1;
```

Comment:
nit: missing a space.

Reply:
Yeah, it's admittedly weird. A couple places in this PR I prioritized minimizing the upstream diff with Android over making our .proto look pretty.